Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.4.0
+FROM ruby:2.7
```

Comment:
Should be 2.6.3: https://github.com/King-County-Equity-Now/decidim-seattle/blob/b3a4fd3f2c50552e0e32b1f37c2f0af0d2bdefe1/.tool-versions